n_cjson_helpers.c

```diff
@@ -149,14 +164,19 @@ JNUMBER JNumberValue(J *item)
     return item->valuenumber;
 }
 
-//**************************************************************************/
 /*!
-    @brief  Return a number from the specified JSON object.
-    @param   rsp The JSON response object.
-    @param   field The field to return.
-    @returns The number found, or 0.0, if not present.
-*/
-/**************************************************************************/
+ @brief Get the floating point value of a number field from a JSON object.
+
+ @param rsp The JSON object.
+ @param field The number field to query.
+
+ @returns The value of the number field if the field exists and is a number and
+          0.0 otherwise.
+
+ @note The returned value is the floating point representation of the number. If
+       the field is an integer rather than a floating point number, use
+       `JGetInt`.
```

Comment:
```suggestion
 @note The returned value is the floating point representation of the number.
 
 @see JGetBool
 @see JGetInt
```
https://www.doxygen.nl/manual/commands.html#cmdsee